### DISCUSSION TITLE
Postal does not release held emails after 60 minutes.

### DISCUSSION THREAD

**Discussion by hugoperes** [postalserver/postal#2389] [Help with using Postal]
I'm using Postal Server version 2.1.2. Upon reaching the sending limit, emails are retained and are not automatically released after 60 minutes.

Postal informs that emails need to be released manually. But this becomes unfeasible when there are many retained emails.

Is there any simple solution for this?

**Comment by willpower232** [comment]
Unfortunately not, just have to open a whole bunch of tabs :grimacing:

**Reply by hugoperes** [reply]
Can you tell me if this was fixed in version 2.14 of the postcard?

**Reply by willpower232** [reply]
Unfortunately no one has added any code to resolve this so it is not fixed currently or in the currently forseeable future.
